client/boot/index.js

```diff
@@ -50,11 +50,11 @@ var config = require( 'config' ),
 	syncHandler = require( 'lib/wp/sync-handler' ),
 	bindWpLocaleState = require( 'lib/wp/localization' ).bindState,
 	supportUser = require( 'lib/user/support-user-interop' ),
-	createReduxStoreFromPersistedInitialState = require( 'state/initial-state' ).default;
+	createReduxStoreFromPersistedInitialState = require( 'state/initial-state' ).default,
+	renderWithReduxStore = require( 'lib/react-helpers' ).renderWithReduxStore;
```

Comment:
From what I tested it also helps when you postpone execution of `require( 'lib/user' )` until it is really needed.